csgo_bot.py

```diff
@@ -102,6 +100,51 @@ def list_mix(message):
     items = db.get_items()
     bot.send_message(chat_id, items, parse_mode='Markdown')
 
+    
+	
+	
+@custom_group_only
+def get_javigon(message):
+	try:
```

Comment:
I think we should follow pep8 identation style [1]

[1] https://www.python.org/dev/peps/pep-0008/#indentation